app/models/authorization_definition.rb

```diff
@@ -22,8 +22,8 @@ def self.all
 
   def editors
     available_forms.select { |form|
-      form.editor.present?
-    }.map(&:editor).uniq(&:id).sort_by(&:name)
+      form.service_provider.present? && form.service_provider.type == 'editor'
```

Comment:
cf mon commentaire sur des méthodes de check des types, ça donnerait 
```suggestion
      form.service_provider.present? && form.service_provider.editor?
```

Reply:
j'ajoute aussi que checker partout la présence c'est pas très élégant

terminons cette PR mais on devrait en discuter parce qu'on a déjà les outils pour faire ça "mieux" (des decorators et le null object pattern FTW!)